lib/attractor/value.rb

```diff
@@ -19,6 +19,10 @@ def current_commit
       history&.first&.first
     end
 
+    def score
```

Comment:
I totally see the intent here and actually think this is a nice improvement, but I'm thinking that the name itself is maybe a bit too general.

In the future, there might be more "scores" that then might need disambiguation - but in fairness, I can't come up with something better at this moment...

Reply:
@julianrubisch Do you want me to revert this?

Reply:
No! I'm just thinking about a better name... give me a few days, I'm kinda swamped here :-)